locales/de.json

```diff
@@ -23,7 +23,7 @@
     "landing": {
       "about": "Über Embloy",
       "head": "Ein Embloy Account - für alle Jobs.",
-      "subHead": "Bewirb dich auf Jobs super einfach direkt mit Embloy.",
+      "subHead": "Verschwende keine Zeit mehr mit Workday-Konten oder langen E-Mail-Threads! Melde dich einmal bei Embloy an, um dich mit einem Klick für Jobs zu bewerben und deine Bewerbungen alle an einem Ort zu verwalten",
```

Comment:
lass das alte hier da

Reply:
oder überleg dir einen satz der kürtzer ist, prägnant ist und sich nach nicht ki generiertem werbe deutsch an hört.-

Reply:
Ging mehr darum, dass in dem "head" eine kurze prägnante Headline steht und im subHead etwas detaillierter (in 1-2 Sätzen) erklärt wird was Embloy aus User Sicht genau macht und wieso man das nutzen sollte. Sich "super einfach direkt" auf Jobs zu bewerben sagt ja nicht wirklich viel aus.